docs/installation/gateways.md

```diff
@@ -0,0 +1,345 @@
+
+At a minimum, your reverse proxy must perform the following tasks:
+
+1. Listen for https traffic on port 443.
+2. Provide a valid root cert or wildcard cert for all subdomains during TLS handshake.
+3. Route all http traffic from port 443 to your gateway-proxy.
+
+### Gateway Proxy
+
+Configure internal network access to your Bee node, and the domain name you are using.
+
+- "BEE_API_URL=http://bee:1633"
+- "BEE_DEBUG_API_URL=http://bee:1635"
+- "HOSTNAME=my-domain.org"
+
+To provide access to ENS Swarm hosted domains and/or CID's as subdomains, enable these too.
+
+- "ENS_SUBDOMAINS=true"
+- "CID_SUBDOMAINS=true"
+
+### A Bee Node
+
+Add links to your Gnosis Chain and Ethereum mainnet RPC providers, or to your own nodes if you are running them.
+
+- "BEE_RESOLVER_OPTIONS=https://mainnet.infura.io/v3/b33b33b33b33b33b33b33b33"
+- "BEE_SWAP_ENDPOINT=https://stake.getblock.io/mainnet/?api_key=b33b33b33b33-b33b33b33b33-b33b33b33b33"
+
+To start up your node and deploy your chequebook without funding with BZZ, you can fund with BZZ later, if you choose to, but it is not needed if your node stays within free usage limits.
+
+- "BEE_SWAP_INITIAL_DEPOSIT=0"
+
+A secure password, to decrypt and encrypt your nodes wallet.
+
+- "BEE_PASSWORD=b55555555555555555b33b33-secure-password"
+
+If using Gateway Proxy facilities which require access to the Bee debug api, such as stamp autobuy.
+
+- "BEE_DEBUG_API_URL=127.0.0.1:1633"
+- "BEE_DEBUG_API_ENABLE=true"
+
+If you would like to see more information about what's happening with your node, increase the verbosity level.
+
+- "BEE_VERBOSITY=debug"
+
+Consider setting your Bee node's address if it has not been automatically connected so that other nodes in the network can discover it.
+
+- "BEE_NAT_ADDR=123.123.123.123:1634"
+
+When you start up your node, check the logs. You will see something like:
+
+```
+cannot continue until there is at least 0.000250 xDAI (for Gas) available on b33b33b33b33b33b33b33b33b33b33b33b33b333
+```
+
+Fund the address with xDAI. Once your have done so, your node should begin to sync postage batch snapshot and once it has done so, connect to the network. 
+
+## Testing your installation
+
+If you have enabled ENS support:
+
+1. https://swarm.my-domain.org should resolve to the Swarm website.
+2. https://my-domain.org/bzz/swarm.eth should resolve to the Swarm website.
+
+### Modes of Operation
+
+Follow the relevant instructions to get your gateway up and running to suit your needs.
+
+###  Personal
+
+Supply the following configuration parameters to Gateway Proxy in order to create an authenticated gateway that automatically stamps chunks with a default stamp.
+
+Limit access to your gateway by providing an authorisation secret known only to you and trusted individuals.
+
+```
+AUTH_SECRET=b33b33b33b33b33b33b33b33b33
```

Comment:
TBH I would rather recommend the `nginx` authentication than the build in one. We wanted to remove it because it is quite limited (there is only one secret for instance that you need to use for all applications).

Reply:
ok, let me see what i can do